packages/next/build/webpack-config.ts

```diff
@@ -38,6 +46,20 @@ export default async function getBaseWebpackConfig (dir: string, {dev = false, d
     .split(process.platform === 'win32' ? ';' : ':')
     .filter((p) => !!p)
 
+  let useTypeScript = false
+  let typeScriptPath = ''
+  let tsConfigPath = path.resolve(dir, 'tsconfig.json')
+
+  if (!isServer) {
+    await verifyTypeScriptSetup(dir)
+    useTypeScript = await exists(tsConfigPath)
+    if (useTypeScript) {
+      typeScriptPath = path.join(
```

Comment:
This should use `resolve` similar what we do for externals

Reply:
`resolve` can't be used here like for externals since it errors when it doesn't exist and we don't want it to error here since we create a default `tsconfig.json` if one doesn't exist

Reply:
Just try/catch the error 😄